code/parachain/runtime/picasso/src/ibc.rs

```diff
@@ -33,21 +33,47 @@ impl DenomToAssetId<Runtime> for IbcDenomToAssetIdConversion {
 	type Error = DispatchError;
 
 	fn from_denom_to_asset_id(denom: &String) -> Result<CurrencyId, Self::Error> {
-		ForeignIbcIcs20Assets::<AssetsRegistry>::from_denom_to_asset_id(denom)
+		log::debug!("converting denom {} to currency id", denom);
+		let denom_bytes = denom.as_bytes().to_vec();
+		if let Some(id) = IbcDenoms::<Runtime>::get(&denom_bytes) {
+			log::debug!("converted {:} to currency id", &id);
+			return Ok(id)
+		}
+
+		// +1 nonce
```

Comment:
What is this comment referring to?

Reply:
can you show him where on IBC?

Reply:
I replaced this trait implementation on the implementation from `ibc.rs`.
The comments aren't relevant already.